3_javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,87 @@
+
+// exercice full_name
+function getFullname(firstname, surname, useFormalName, isWomen ){
+
+    if( firstname === "" & surname === ""){
+        return "firstname and surname are missing"
+    }
+    if (useFormalName === true){
+        if (isWomen === true){
+            return `lady ${firstname} ${surname}`;
+        } else{
+            return `Lord ${firstname} ${surname}`;
+        }    
+    } else {
+        return `${firstname} ${surname}`;
+    }
+}
+
+const fullName = getFullname("Benjamin", "Hughes",true);
+console.log(fullName);
+let fullName1 = getFullname("nada", "jrad") ;
+let fullName2 = getFullname("jannah", "jrad") ;
+console.log(fullName1);
+console.log(fullName2);
+
+// event day 
+function getEventWeekday(numberOfTheDay){
+    let today = new Date();
+    let day = today.getDay();//1,2,3...
+    let weekdays = ["Sunday", "Monday", "Tuesday", "Wednesday" , "Thursday", "Friday", "Saturday"];
+   
+    return weekdays[(day + numberOfTheDay) % 7];
+}
+
+console.log(getEventWeekday(0));
+console.log(getEventWeekday(3));
+console.log(getEventWeekday(9));
+
+// weather
+function weather(tempertaure){
+    if(tempertaure >= 20 ){
+        return "wear shorts and dress";
+    }  if (tempertaure <= 10 ){
+        return "wear  a coat ";
+    } else {
+        return "wear jacket jeans";
+    }
+}
+let  clothesToWear = weather(8);
+console.log(clothesToWear);
+// Student manager
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (class07Students.length >= 6){
+       if(studentName === "Queen"){
+        class07Students.pop();
```

Comment:
Nice work. Although I think the Queen does not need to remove anyone to fit in. :) Also consider logging the return values because this way the user does not get feedback on what happened.

Reply:
@szoboszlaianna thanks a lot for the review and your feedback, i really appreciate it and it helps me to get better 🙏